jenkins_jobs/trigger_host_os_iso_build_from_builds_repo.groovy

```diff
@@ -0,0 +1,48 @@
+job('trigger_host_os_iso_build_from_builds_repo') {
+  label('!master')
+  logRotator {
+    numToKeep(30)
+  }
+  concurrentBuild()
+  throttleConcurrentBuilds {
+    maxPerNode(1)
+  }
+  parameters {
+    stringParam('sha1', '', 'SHA-1 of the commit to build.')
+  }
+  properties {
+    githubProjectUrl("https://github.com/${GITHUB_ORGANIZATION_NAME}/builds/")
+  }
+  triggers {
+    githubPullRequest {
+      userWhitelist("${GHPRB_ADMIN_USER}")
+      orgWhitelist("${GHPRB_ADMIN_ORGANIZATION}")
+      allowMembersOfWhitelistedOrgsAsAdmin()
+      cron('H/5 * * * *')
+      triggerPhrase('.*start\\W+(tests|build_iso).*')
+      extensions {
+	commitStatus {
+	  context('Build Host OS')
+	}
+      }
+    }
+  }
+  steps {
+    downstreamParameterized {
+      trigger('build_host_os_iso') {
+	block {
+	  buildStepFailure('FAILURE')
+	  failure('FAILURE')
+	  unstable('UNSTABLE')
+	}
+	parameters {
+	  predefinedProps([BUILDS_REPO_COMMIT: '$sha1',
+			   VERSIONS_REPO_COMMIT: 'master'])
```

Comment:
Need to add the script from the other trigger:

https://github.com/open-power-host-os/infrastructure/blob/master/jenkins_jobs/trigger_host_os_iso_build_from_versions_repo.groovy#L36

And the credentials binding plugin:

https://github.com/open-power-host-os/infrastructure/blob/master/jenkins_jobs/trigger_host_os_iso_build_from_versions_repo.groovy#L53